webpack.config.babel.js

```diff
@@ -230,7 +229,7 @@ const baseConfig = {
     setImmediate: false,
   },
 
-  devtool: PRODUCTION_BUILD ? 'source-map' : undefined,
+  devtool: PRODUCTION_BUILD ? 'source-map' : 'cheap-eval-source-map',
```

Comment:
What's the difference between these 2 source map devtools?

Reply:
This configuration builds the full sourcemaps for production. For development we were not generating sourcemaps and this was printing a lot of warnings in the browser console. When using `eval-cheap-source-map`, the sourcemap would be a base64 that contains only line data and no column mappings. This results in a faster build compared to a production build where the full sourcemap is generated.